src/pages/homePage/molecules/subHeaderCategories/SubHeaderCategories.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+// style
+import './subHeaderCategories.scss';
+
+const SUB_HEADER_CATEGORIES = ['Vegetables & Fruits', 'Dairy & Breakfast', 'Muchies', 'Cold Drinks & Juices', 'Instant & Frozen Food',
+    'Tea, Coffee & Health Drinks', 'Bakery & Biscuits'];
+const EXTRA_CATEGORIES = [{
```

Comment:
Why are these constants , we have to make a constant file at this molecule level and import it from here.